cookbooks/aws-parallelcluster-slurm/templates/default/slurm/slurm.conf.erb

```diff
@@ -20,6 +20,9 @@ SwitchType=switch/none
 SlurmctldPidFile=/var/run/slurmctld.pid
 SlurmdPidFile=/var/run/slurmd.pid
 ReconfigFlags=KeepPartState
+<% if node['cluster']['enable_nss_slurm'] == 'true' -%>
+LaunchParameters=enable_nss_slurm
```

Comment:
if not done already we need to call this out explicitly in changelog ince there might be users adding LaunchParameters in Slurm config already